Airports/EGSX/SMR/Regions.txt

```diff
@@ -0,0 +1,2500 @@
+N051.43.10.590 E000.09.07.006
+N051.43.11.006 E000.09.05.967
+N051.43.11.235 E000.09.05.333
+N051.43.11.497 E000.09.04.681
+N051.43.11.675 E000.09.04.236
+N051.43.11.954 E000.09.03.461
+N051.43.12.269 E000.09.02.643
+N051.43.12.488 E000.09.02.058
+N051.43.12.622 E000.09.01.691
+N051.43.12.801 E000.09.01.235
+N051.43.12.982 E000.09.00.717
+N051.43.13.123 E000.09.00.399
+N051.43.13.371 E000.08.59.687
+N051.43.13.944 E000.08.58.199
+N051.43.14.290 E000.08.57.261
+N051.43.14.343 E000.08.57.114
+N051.43.14.379 E000.08.57.171
+N051.43.14.419 E000.08.57.276
+N051.43.14.474 E000.08.57.378
+N051.43.15.046 E000.08.56.861
+N051.43.15.008 E000.08.56.646
+N051.43.14.988 E000.08.56.478
+N051.43.14.987 E000.08.56.459
+N051.43.14.984 E000.08.56.100
+N051.43.14.964 E000.08.55.761
+N051.43.14.945 E000.08.55.574
+N051.43.15.019 E000.08.55.313
+N051.43.15.332 E000.08.54.479
+N051.43.15.430 E000.08.54.180
+N051.43.15.862 E000.08.53.085
+N051.43.16.280 E000.08.51.929
+N051.43.16.793 E000.08.50.648
+N051.43.17.365 E000.08.49.105
+N051.43.17.749 E000.08.48.140
+N051.43.18.322 E000.08.46.602
+N051.43.18.657 E000.08.45.735
+N051.43.19.300 E000.08.44.064
+N051.43.19.651 E000.08.43.139
+N051.43.20.183 E000.08.41.705
+N051.43.20.814 E000.08.40.001
+N051.43.20.957 E000.08.39.809
+N051.43.21.048 E000.08.39.643
+N051.43.21.162 E000.08.39.537
+N051.43.21.292 E000.08.39.447
+N051.43.21.415 E000.08.39.409
+N051.43.21.557 E000.08.39.369
+N051.43.21.730 E000.08.39.418
+N051.43.21.912 E000.08.39.441
+N051.43.22.050 E000.08.39.524
+N051.43.19.386 E000.08.36.938
+;Disused
+REGIONNAME North Weald
+smrGDrunway N051.43.05.714 E000.09.15.236
+N051.43.05.253 E000.09.16.554
+N051.43.04.856 E000.09.17.502
+N051.43.03.533 E000.09.16.960
+N051.43.01.143 E000.09.23.343
+N051.43.00.210 E000.09.25.746
+N051.42.59.620 E000.09.25.146
+N051.42.59.561 E000.09.25.116
+N051.42.59.236 E000.09.24.668
+N051.42.58.862 E000.09.24.580
+N051.42.58.862 E000.09.24.933
+N051.42.58.850 E000.09.25.454
+N051.42.58.826 E000.09.26.207
+N051.42.58.815 E000.09.27.096
+N051.42.58.760 E000.09.27.743
+N051.42.58.567 E000.09.28.397
+N051.42.59.328 E000.09.29.619
+N051.42.59.200 E000.09.30.014
+N051.42.58.444 E000.09.32.060
+N051.42.58.036 E000.09.33.108
+N051.43.00.687 E000.09.35.181
+N051.43.01.061 E000.09.34.900
+N051.43.01.334 E000.09.34.610
+N051.43.01.668 E000.09.34.013
+N051.43.02.058 E000.09.33.031
+N051.43.02.435 E000.09.32.111
+N051.43.02.688 E000.09.31.451
+N051.43.03.202 E000.09.31.934
+N051.43.04.038 E000.09.32.583
+N051.43.05.450 E000.09.29.080
+N051.43.07.866 E000.09.22.631
+N051.43.08.896 E000.09.19.809
+N051.43.09.061 E000.09.19.445
+N051.43.07.440 E000.09.18.680
+N051.43.06.474 E000.09.18.247
+N051.43.06.354 E000.09.18.165
+N051.43.06.986 E000.09.16.511
+N051.43.07.177 E000.09.15.990
+N051.43.07.474 E000.09.15.169
+N051.43.07.577 E000.09.14.818
+N051.43.07.778 E000.09.14.333
+N051.43.07.966 E000.09.13.801
+N051.43.08.172 E000.09.13.288
+N051.43.08.414 E000.09.12.650
+N051.43.06.966 E000.09.11.976
```

Comment:
2500 lines for an SMR for an airport which is not in the AIP is quite an intense amount of data everyone has to load each time we boot Euroscope.

Can I suggest removing some less relevant areas (e.g. all the SouthEast corner which seems to be covered in old ambulances) to help preserve the high quality data in other areas.